### PR TITLE
Feature/31 update to current 5x remove form from build

### PR DESCRIPTION
**Pull request by notmyown**

### CODE REVIEW

**Comment by notmyown** [comment]
since the dogu is not depending on the swagger-ui fork anymore, the custom cloudogu styling would be lost. Therfore a new css file was added to the index template of the swagger-dogu.

By analysing the SCSS Rules and mixins, all compiled css classes are configured to use the custom cloudogu colors

grafik

so the official swagger-ui archive can be used along with the custom cloudogu theme

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- dismiss special fork from swagger-ui (#31)
```
Comment (ppxl) [inline CHANGELOG.md:9]: yes! :muscle:

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,10 @@
 FROM registry.cloudogu.com/official/base:3.20.2-1 as swaggerui
```
Comment (ppxl) [inline Dockerfile:1]: This image could be newer with fewer CVEs.
```suggestion
FROM registry.cloudogu.com/official/base:3.20.3-2 as swaggerui
```

File: Dockerfile [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+ENV SWAGGERUI_VERSION=5.17.14 \
+    SWAGGERUI_ZIP_SHA256="481244d0812097b11fbaeef79f71d942b171617f9c9f9514e63acbe13e71ccdc"
 RUN apk update && apk add curl
-RUN curl -Lsk --fail --silent --location --retry 3 https://github.com/cloudogu/swagger-ui/releases/download/v${SWAGGERUI_VERSION}/swagger-ui-${SWAGGERUI_VERSION}.zip -o /tmp/swagger-ui.zip
+RUN curl -Lsk --fail --silent --location --retry 3 https://github.com/swagger-api/swagger-ui/archive/refs/tags/v${SWAGGERUI_VERSION}.zip -o /tmp/swagger-ui.zip
 RUN echo "${SWAGGERUI_ZIP_SHA256} */tmp/swagger-ui.zip" | sha256sum -c -
-RUN unzip /tmp/swagger-ui.zip -d /tmp
+RUN unzip /tmp/swagger-ui.zip -d /tmp && mv /tmp/swagger-ui-${SWAGGERUI_VERSION}/dist /tmp/dist
 
 FROM registry.cloudogu.com/official/base:3.20.2-1
```
Comment (ppxl) [inline Dockerfile:9]: As above: 
```suggestion
FROM registry.cloudogu.com/official/base:3.20.3-2
```

File: docs/development/local_setup_de.md [hunk 4]
```diff
@@ -24,3 +24,11 @@ zip -r swagger-ui-4.9.0-1.zip dist
 sha256sum swagger-ui-4.9.0-1.zip > swagger-ui-4.9.0-1.zip.sha256
 3. Dem Release anfügen https://github.com/cloudogu/swagger-ui/releases
 ```
+
+(ab 5.17.14)
+```
+Die Releases werden wieder über die offizielle Seite von swagger-ui gepflegt.
+
+Im Dockerfile des Gogus wird nur noch die entsprechende Version angepasst.
```
Comment (ppxl) [inline docs/development/local_setup_de.md:32]: Typo:
```suggestion
Im Dockerfile des Dogus wird nur noch die entsprechende Version angepasst.
```

File: docs/development/local_setup_en.md [hunk 5]
```diff
@@ -23,3 +23,14 @@ zip -r swagger-ui-4.9.0-1.zip dist
...
+
+### Setup since 5.17.14-1
+Releases can be found on the official swagger-ui download page.
+The fork will be dismissed.
+```
+1. download artefact from swagger-ui
+2. generate the checksum 
+sha256sum swagger-ui.zip
+3. Edit Dockerfile for new version
+```
```
Comment (ppxl) [inline docs/development/local_setup_en.md:35]: This content does not seem like a literal translation of the german variant. Why the difference?

File: dogu.json
Comment (ppxl) [inline dogu.json:1]: `100644 → 100755 `
Is there are reason for the general setting of the executable bit in all files of this change? There aren't even executable files among this change set.

**Review by ppxl — CHANGES_REQUESTED** [review]
Hi, thanks for the great work! I found only some minor issues. Anyhow, there seem to exist some issues with the unix permissions because all files received the executable bit which is not quite ideal.

**Comment by ppxl** [comment]
LGTM, thanks for the work :blue_heart:

**Pull request merged**